weather_mv/loader_pipeline/bq.py

```diff
@@ -87,7 +87,7 @@ class ToBigQuery(ToDataSink):
     """
     output_table: str
     variables: t.List[str]
-    area: t.Tuple[int, int, int, int]
+    area: t.List[float]
```

Comment:
This LGTM. A quick question: is there early validation to check that users must specify 4 floats?

Reply:
Yes, there is [validation](https://github.com/google/weather-tools/blob/e67f361899aebe6efaac23628cbe75bd68ce31c5/weather_mv/loader_pipeline/bq.py#L133-L135) that user must specify 4 values for `--area`.

Reply:
Thanks, perfect.